### PR TITLE
RHDEVDOCS-3177 Tracker for 1861266 - [DOCS] New USE_JAVA_VERSION vari…

### PR DESCRIPTION
**Pull request by rolfedh**
…able not documented

- Aligned team: Dev Tools
- For branches: 4.6+
- Jira: https://issues.redhat.com/browse/RHDEVDOCS-3177
- Direct link to doc preview: https://50560--docspreview.netlify.app/openshift-enterprise/latest/cicd/jenkins/images-other-jenkins-agent.html#images-other-jenkins-agent-env-var_images-other-jenkins-agent
- SME review: @akram (approved)
- QE review: @jitendar-singh (approved)
- Peer review: @jc-berger (approved)
- All reviews complete. Please merge now.

### CODE REVIEW

File: modules/images-other-jenkins-agent-env-var.adoc [hunk 1]
```diff
@@ -50,4 +50,8 @@ JVM threads.
 |Specifies additional options for the Jenkins JVM. These options are appended to all other options, including the Java options above, and can be used to override any of them, if necessary. Separate each additional option with a space and if any option contains space characters, escape them with a backslash.
 |Example settings: `-Dfoo -Dbar`; `-Dfoo=first\ value -Dbar=second\ value`
 
+|`USE_JAVA_VERSION`
+|Specifies the version of Java to inject into the container agent. The default value is `java-11`.
```
Comment (akram) [inline modules/images-other-jenkins-agent-env-var.adoc:54]: The more exact wording would be "Specifies the version of Java version to use to run the agent in its container. The default value is `java-11`. Possible alternative value is `java-1.8.0`. If you extend the container base image, you can specify any alternatives version of java using it associated suffix".

@rolfedh it follows the pattern explained here 
https://access.redhat.com/documentation/en-us/openjdk/8/html-single/installing_and_using_openjdk_8_for_rhel/index

**Review by akram — CHANGES_REQUESTED** [review]
The more exact wording would be "specify which java version to use to run the agent".

**Comment by rolfedh** [comment]
Issues with the generated preview. See following screenshot instead.
Screenshot from 2022-09-21 14-48-45

**Review by jc-berger — APPROVED** [review]
/lgtm

I consulted the openshift checklist when conducting this peer review: https://docs.google.com/document/d/1WN7k72PKxcPA__erp5TK-CefmS_sVwE3r40PvpmNk3E/edit#heading=h.mgz8v4w1kg86

**Pull request merged**